python/activator/activator.py

```diff
@@ -70,8 +70,15 @@
 
 # Initialize middleware interface; TODO: we'll need one of these per detector.
 repo = f"/tmp/butler-{os.getpid()}"
+central_butler = Butler(calib_repo,
+                        # TODO: How do we get the appropriate instrument name
```

Comment:
Why is `config_instrument` incorrect?  Or if there needs to be a lookup table, that's OK, as there is a well-defined list of supported instruments.

Reply:
This has to do with some UIs requiring a class name and others an instrument short name. As mentioned on Slack, I would just create a global `Instrument` object and then use that for the translation.

Reply:
I'm going to leave this question to the "reintegrate activator and interface" ticket (that I guess doesn't exist?), if that's ok? We could create an `Instrument` here, but we'd still have to change the dict in `uploader.py` to have full class names, I think? And we'd also have to expand the eups table to have all the correct obs packages. I feel like that's enough moving parts that it should be a separate ticket.

Reply:
I don't see any reason why we'd have to change `uploader.py`; we'd just need to make sure _this file_ is always keeping straight when it wants the class name and when it wants the short name.